tools/trap/ramdumpParser.py

```diff
@@ -600,7 +600,7 @@ def find_crash_point(log_file, elf):
                     continue
 
             # Get the assert location PC value
-            if 'up_assert: Assertion failed at file:' in line:
+            if 'print_assert_detail: Assertion failed at file:' in line:
```

Comment:
How about checking both? or ignoring prefix?
Important (necessary) sentence is after `:`.
If we change the function name, we should modify this again. But it's meanless message in the TRAP.

Reply:
@thapav Could you let me know ETA for this?

Reply:
Yes your suggestions is a good one. I have incorporated the changes.